### ISSUE TITLE
History and Rollback : Don't display details of last deployed revision when opening the screen.

### ISSUE THREAD

**Issue by lchastel** [argoproj/argo-cd#12183]
# Summary
In History and Rollback, the details on last deployed revision are displayed.
I would like the last deployed revision to be displayed as a summary (like the other ones) when the window is opened.

The user could click on it to have the details.

# Motivation
I would like to avoid to scroll to the bottom of the page to see previous deployed revisions.
It will be more readable.

# Proposal
Display the last deployed version like the other ones.

I attached a screenshot of an application with 2 deployed devisions.
I would like the latest revision to be displayed has the second one.
![01](https://user-images.githubusercontent.com/44160341/215142034-3d868e90-8736-4709-9d30-952786bcdb4c.png)
![02](https://user-images.githubusercontent.com/44160341/215142045-1272d107-3b6b-4805-82c1-60bf9010cc75.png)

**Comment by alexef** [comment]
We have the same issue that the expanded revision details takes too much screen space, making it hard to scroll and find a previous version to rollback.

However, every time this happens, the issue is that *all parameters* are displayed (not only the overrides).

May I suggest a different solution? Either collapse parameters by default and require an extra click to inspect them OR in Revision and History view show only overriden parameters. WDYT @saumeya ?

**Comment by saumeya** [comment]
Just looked at the behavior, yes it makes sense to have a default collapsed section and a button to inspect the revision further (if we click on the Revision number it does expand it (however it opens a new tab to the commit)). 
It would be better to have a separate button to collapse and expand all the details.
What are the overridden parameters @alexef, the image tag that was revised from the last revision? But it is maybe good to have all parameters visible

**Comment by alexef** [comment]
@saumeya in the first screenshot above, the two parameters prefixed with 🔨 are `branch` and `env`. All other parameters have the default value, and I wouldn't show them if possible.

**Comment by saumeya** [comment]
Thanks @alexef , got it. That will be specifically for helm parameters. I am ok with olny showing the overridden parameters or helm could also have a collapsible section as well.

**Comment by tukak** [comment]
It would be great to see this implemented - we have helm applications with hudreds of parameters and finding the right revision to rollback to is very painful.

**Comment by drileyst** [comment]
Yeah having the first revision collapsed would be a huge improvement.

**Comment by Tchoupinax** [comment]
Hey,
I implemented a display button [here](https://github.com/argoproj/argo-cd/pull/16871). Give us your feedback!
